tst/schedulers/bayesopt/test_hp_ranges.py

```diff
@@ -12,6 +12,7 @@
 # permissions and limitations under the License.
 
 from collections import Counter
+from distutils.command.config import config
```

Comment:
Where do we we need this one?

Reply:
My IDE added this without me noticing it :D Will remove.